### PR TITLE
update readme to reflect github PAT creation

### PR DESCRIPTION
**Pull request by joshuahiggins**
Take away a little of the guesswork for people with token exporting and such.

### CODE REVIEW

File: bunfig.toml [hunk 1]
```diff
@@ -1 +1,4 @@
-framework = "next"
```
Comment (lonbes) [inline bunfig.toml:1]: While the idea is right, the project directory bunfig.toml will need to have `framework = "next"`, as this is what lets `bun bun` do it's thing.

See previous file comment.

File: README.md [hunk 2]
```diff
@@ -9,14 +9,19 @@ We're believers in [BunJS](https://github.com/oven-sh/bun) and utilize it for mo
 Bun manages memory differently than Node, and is developed to run primarily on *nix based systems (including MacOS). While a native implementation of Bun for Windows is on their roadmap, one is not available.
 
 ### Bun Config
-Since our package registry is hosted publicly on Github, Bun is going to need a little bit of configuration to download our packages. 
+Since our package registry is hosted publicly on Github, Bun is going to need a little bit of configuration to download our packages.
 
-Create a new Github Personal Access Token (PAT) with repository read access.
+#### Github Repo Access
 
-In your `home` directory, create a new file called `.bunfig.toml` and insert the following:
```
Comment (lonbes) [inline README.md:16]: This file will need to be created in the **user's** home directory. (`~/`), you have the right understanding though! Bun will look at the project directory config first, then the home.

File: bunfig.toml (hunk 1 above)
Comment (joshuahiggins) [reply bunfig.toml:1]: It's still present on line 4. It's just moved down since Bun was failing to run with it above the `install.scopes` block.

**Review by lonbes — CHANGES_REQUESTED** [review]
Please address comments. THANK YOU FOR BEING OUR FIRST CONTRIBUTOR!

File: bunfig.toml (hunk 1 above)
Comment (lonbes) [reply bunfig.toml:1]: Oh wait Josh I am a dingus, you are correct. Not everyone will want to globally configure, best to keep it project oriented.

**Pull request merged**